test/integration/search_test.rb

```diff
@@ -7,11 +7,36 @@ class SearchTest < ActionDispatch::IntegrationTest
   end
 
   should "allow us to embed search results in an iframe" do
-    stub_request(:get, %r[#{Plek.new.find('search')}/search.json*])
-      .to_return(body: JSON.dump(results: [], facets: []))
+    stub_any_rummager_search_to_return_no_results
 
     get "/search?q=tax"
 
     assert_equal "ALLOWALL", response.headers["X-Frame-Options"]
   end
+
+  context "given an unscoped search" do
+    should "display the correct results" do
+      stub_unscoped_search("sandwiches")
+      visit "/search?q=sandwiches"
+      assert page.has_content? "Offshore sandwich device"
+      assert page.has_content? "flood risk in Sandwich"
+    end
+  end
+
+  context "given a search scoped to a manual" do
+    setup do
+      stub_search_to_return_results_scoped_to_manual
+      stub_scope_object
+      stub_unscoped_rummager_request
```

Comment:
I wasn't quite sure why we need to stub an unscoped rummager request in a test for a scoped search, but it appears to be needed. Good to have this spelled out more 👍